### PR TITLE
Modity start-up doc for v1.3.0

### PR DESCRIPTION
**Pull request by chivalryq**
Signed-off-by: qiaozp <chivalry.pp@gmail.com>

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -19,11 +19,20 @@ You'll get to know:
 - Get familiar with core concepts as [Application](./getting-started/core-concept#application), [Environment](getting-started/core-concept#environment) and [Target](getting-started/core-concept#target)
 - Finished an application delivery by operating VelaUX
 
+## Get password and login
```
Comment (barnettZQG) [inline docs/quick-start.md:22]: advise write it in the install doc.

File: docs/install.mdx [hunk 2]
```diff
@@ -286,7 +286,7 @@ If you are helm user, you can also use helm to install kubevela core:
 ```
 helm repo add kubevela https://charts.kubevela.net/core
 helm repo update
-helm install --create-namespace -n vela-system kubevela kubevela/vela-core --version 1.2.4 --wait
+helm install --create-namespace -n vela-system kubevela kubevela/vela-core --version 1.3.0-beta.2 --wait
```
Comment (barnettZQG) [inline docs/install.mdx]: change install the velaux addon command:
```
vela addon enable velaux --version 1.3.0-beta.2
```

**Comment by barnettZQG** [comment]
update the Chinese doc.

File: docs/quick-start.md (hunk 1 above)
Comment (chivalryq) [reply docs/quick-start.md:22]: Write it there too

File: docs/quick-start.md [hunk 3]
```diff
@@ -19,11 +19,20 @@ You'll get to know:
...
+
+After install VelaUX, you need to log in. Default username is "admin" and you can get the password using command below
+```shell
+vela logs -n vela-system --name apiserver addon-velaux | grep "init admin user"
```
Comment (barnettZQG) [inline docs/quick-start.md]: It's can not work in beta.2 version.

Comment (chivalryq) [reply docs/quick-start.md]: Let's hold this until we publish a new version?

File: docs/install.mdx [hunk 4]
```diff
@@ -316,6 +316,12 @@ Choose `> Cluster: local | Namespace: vela-system | Component: velaux | Kind: Se
 
 If you have loadbalaner or ingress, please refer to [VelaUX addon docs](./reference/addons/velaux) for more advanced installation ways.
 
+VelaUX need authentication. Default username is admin and get default password as below
+
+```shell
+vela logs -n vela-system --name apiserver addon-velaux | grep "init admin user"
```
Comment (wonderflow) [inline docs/install.mdx]:
```suggestion
vela logs -n vela-system --name apiserver addon-velaux | grep "initialized admin username"
```

Comment (wonderflow) [reply docs/install.mdx]: changed in https://github.com/oam-dev/kubevela/pull/3563/files

File: docs/quick-start.md (hunk 3 above)
Comment (wonderflow) [reply docs/quick-start.md]: changed to `initialized admin username` https://github.com/oam-dev/kubevela/pull/3563/files

**Review by wonderflow — APPROVED** [review]

**Pull request merged**